doc/src/build/programming-with-objects/ch1-object-basics.md

```diff
@@ -159,7 +159,7 @@ This will tell you the current client address.
 
 First, we need to publish the code on-chain. Assuming the path to the root of the repository containing Sui source code is $ROOT:
 ```
-$ sui client publish --path $ROOT/sui_programmability/examples/objects_tutorial --gas-budget 10000
+$ sui client publish --path $ROOT/sui/sui_programmability/examples/objects_tutorial --gas-budget 10000
```

Comment:
I don't think there is `sui` in-between?

Reply:
I think I found someone without it and tried to fix it, but fixed it incorrectly.